docs/antctl.md

```diff
@@ -666,3 +667,29 @@ worker1 172.18.0.4 Alive
 worker2 172.18.0.3 Alive 
 worker3 172.18.0.2 Dead
 ```
+
+### Upgrade existing resources
+
+Antctl supports upgrading existing resources of CRDs managed by Antrea to the latest version. The related sub commands
+should be run out-of-cluster.
+
+This command is used to upgrade all existing persisted resources to the latest API version:
+
+```bash
+antctl upgrade api-storage
+```
+
+This command is used to upgrade persisted resources for Antrea CRD `antreaagentinfos.crd.antrea.io` to the latest API
```

Comment:
persisted resources for Antrea CRD `antreaagentinfos.crd.antrea.io` ->  AntreaAgentInfo resources

Reply:
Done.